public/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-  "short_name": "Web Interface",
-  "name": "Create React Web Interface",
+  "short_name": "React App",
```

Comment:
Why did you change name?

Reply:
Because i create all wat u see in this branch how a new project. All files in old web-interface1 branch are useless. We can use any application name.What name do you prefer? :)

Reply:
Would be nice if you will name it. It;s not a blank project, not a testing. It definataly a working project must be named

Reply:
ok :)